lib/charms/layer/jenkins/configuration.py

```diff
@@ -11,8 +12,47 @@
 class Configuration(object):
     """Manage global Jenkins configuration."""
 
+    @staticmethod
+    def _set_prefix(prefix):
```

Comment:
Please move private methods at the bottom, after public ones, since people reading the code are typically interested in the public APIs, at least at first.